### ISSUE TITLE
Copying sparse files?

### ISSUE THREAD

**Issue by jo-erlend** [astrada/google-drive-ocamlfuse#748]
I've tried using cp --sparse=always when copying a file to my Google Drive, but that doesn't work. It copies, but it copies the whole thing. Do you think it would be possible to make it work, or is there a limitation in the underlying system?

**Comment by astrada** [comment]
No, it's not possible as the Drive API doesn't support that feature.
